### PR TITLE
Be user details

### PR DESCRIPTION
**Pull request by eeels22**
# Changes proposed

- Add description field and its getter and setter method to User (max length 1000 characters)
- Add PUT mapping to update user description in UserController

## Question
- I did not call UserService but do you think I should refactor? The reason for this is the Controller already has a repository.

## User story
`As a user of Cuddly, I want to fill up my user detail, so that I can display my information`

Screenshot 2021-05-07 at 13 50 03

### CODE REVIEW

**Review by janko101 — APPROVED** [review]
It looks good to me

File: src/main/java/se/kth/sda/skeleton/user/UserController.java [hunk 1]
```diff
@@ -18,11 +15,13 @@
 
     private UserRepository userRepository;
     private AuthService authService;
+    private UserService userService;
 
     @Autowired
     public UserController(UserService userService, UserRepository userRepository, AuthService authService) {
         this.userRepository = userRepository;
         this.authService = authService;
+        this. userService = userService;
```
Comment (NeuralAlchemist) [inline src/main/java/se/kth/sda/skeleton/user/UserController.java]: doesn't seem useful as its not really used in the controller. you can either remove it in line 24 and 21 and 18 or make sure that there is no space after . in line 24

**Review by NeuralAlchemist — COMMENTED** [review]
please check this out

File: src/main/java/se/kth/sda/skeleton/user/UserController.java (hunk 1 above)
Comment (eeels22) [reply src/main/java/se/kth/sda/skeleton/user/UserController.java]: Thanks, agree. Have removed it!

**Review by NeuralAlchemist — APPROVED** [review]
Thanks!

**Pull request merged**